apps/core/src/hooks/useGetTransferAmount.ts

```diff
@@ -26,26 +29,24 @@ export function useGetTransferAmount(txnData: SuiTransactionBlockResponse) {
                               ? owner.ObjectOwner
                               : '',
                       amount:
-                          coinType === SUI_TYPE_ARG && BigInt(amount) < 0n
-                              ? BigInt(amount) + BigInt(gas ?? 0n)
+                          BigInt(amount) < 0n
```

Comment:
I see that it was checking to be less than 0 but do you know why? Based on my understanding we add the gas cost to the whole amount. Why only when it is less than 0?

Also do we know that amount is in SUI? otherwise we cannot add gas to it

Reply:
1 - Added back the check for SUI 
2 - This is for negative balance changes which don't include gas, so we add it back, and take the absolute value for displaying the total "sent" 

That being said we have some new designs coming for displaying balance changes which I'll be working on this sprint, so this will all change. Basically was trying to bridge the gap between then and now so we could have some kind of display for amounts back in wallet

Reply:
1. Where is the check for SUI?
2. Yeah we definitely need to change this. `-(BigInt(amount) + gas)` doesn't mean it will always be >0 there is a chance abs(gas) > abs(amount).